src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
```

Comment:
please use destructuring according to checklist.md
![image](https://github.com/mate-academy/react_person/assets/91946036/c30b4b60-2e1c-4785-bfac-aa7a905f2588)
![image](https://github.com/mate-academy/react_person/assets/91946036/d9daf538-51e9-40a9-ad60-ff8c5e848b7f)